govcd/externalnetwork_test.go

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
Consider adding the necessary tags here
```
// +build extnetwork network functional ALL

```
This way, after the PR is merged, it will be ready to run.
The drawback is that this test won't run with `make test` until the PR is merged. 

Having the `network` tag means that the test will run with the rest of the networking. The `extnetwork` tag will also allow running the test in isolation, if we want to try the creation of an external network on a non-configured vCD.

Reply:
I merged with latest PR and added tags.